lib/main.dart

```diff
@@ -80,13 +82,13 @@ class _MyHomePageState extends State<MyHomePage> {
     ''';
 
     MutationOptions mutationOptions = MutationOptions(
-      document: gql(startPersonalWishlistMutations),
-      variables: const <String, dynamic>{
-        'dto': {
-          'firstMessageText': 'Gaming mechanical keyboard',
-          'type': 'Product'
-        },
-      }
+        document: gql(startPersonalWishlistMutations),
+        variables: const <String, dynamic>{
+          'dto': {
+            'firstMessageText': 'Gaming mechanical keyboard',
```

Comment:
Shouldn't first message be what are you looking for? @AsTroLog08 how is it implemented in web client?